wrappers/ortools.rb

```diff
@@ -416,7 +416,7 @@ def solve(vrp, job, thread_proc = nil, &block)
           type: relation.type.to_s,
           linked_ids: current_linked_ids,
           linked_vehicle_ids: current_linked_vehicles,
-          lapse: relation.lapse || -1
```

Comment:
If I am not mistaken we decided to keep the default -1 inside ortools https://github.com/Mapotempo/optimizer-ortools/pull/20 Do I misremember? If not, this line will break the "off" logic, right?

Reply:
You are right, we talked about something like that. But here we know that relation.lapse will never be nil. Do we still want to check ? 

I think this can indeed be useful in the case we generate a relation within algorithm that would need a lapse. But at the same time, if we do not provide a lapse then we will have a problem during development phase and end up providing a lapse.